contrib/pg_tde/src/pg_tde_event_capture.c

```diff
@@ -76,7 +76,7 @@ checkEncryptionClause(const char *accessMethod)
 		if (principal_key == NULL)
 		{
 			ereport(ERROR,
-					(errmsg("failed to retrieve principal key. Create one using pg_tde_set_principal_key before using encrypted tables.")));
+					(errmsg("failed to retrieve principal key. Create one using pg_tde_set_server_principal_key before using encrypted tables.")));
```

Comment:
This function is used by multiple places, most of the time the current error message is correct - wal is a special case.

Reply:
Hahah, wow, I change the wrong instace.

Reply:
Fixed that, but will look at all instances and see if any of them can be accessed in the WAL code path.